modules/router/manifests/assets_origin.pp

```diff
@@ -64,6 +64,11 @@
       logstream => present;
   }
 
+  file { '/var/www/googlec908b3bc32386239.html':
+    ensure  => present,
+    content => template('router/googlec908b3bc32386239.html.erb'),
```

Comment:
Do we know if the value here should be secret, or can be public?

Reply:
I think it needs to be public so Google can access it

Reply:
I know it needs to be publicly accessible, but I'm wondering if it should be publicly discoverable in the govuk-puppet configuration.

Reply:
I don't know if there's another way to do this. What would be the issue of having it here?

Reply:
The alternative would be to use the hieradata in the govuk-secrets repository. But if we're confident that it's OK to have the filename and contents in the clear, then it's fine to keep it here.